libgalois/include/katana/Executor_Deterministic.h

```diff
@@ -192,7 +198,7 @@ class ReaderContext : public katana::UnionFindNode<ReaderContext>,
 
   bool propagate() { return this->find()->isReady(); }
 
-  virtual void alwaysAcquire(Lockable*, katana::MethodFlag) {
+  void alwaysAcquire(Lockable*, katana::MethodFlag) override {
     KATANA_DIE("unreachable");
```

Comment:
If this is unreachable then it probably shouldn't even be a method on this class. Can it be moved out of the base class indo the "mutable contexts" branch of the heirarchy?

Also, `KATANA_LOG_FATAL`?